js/views/reviewsVw.js

```diff
@@ -0,0 +1,103 @@
+var __ = require('underscore'),
+    Backbone = require('backbone'),
+    $ = require('jquery'),
+    RatingCl = require('../collections/ratingCl'),
+    baseVw = require('./baseVw'),
+    ReviewView = require('./reviewVw');
+
+module.exports = baseVw.extend({
+
+  className: "flexCol-12 flex-border js-tabTarg js-reviews hide",
+
+  events: {
+  },
+
+  VIEWS_PER_BATCH: 25,
+
+  MAX_VIEWS: 1500,
+
+  initialize: function(options) {
+    var self = this;
+
+    this.options = options || {};
+
+    if (!options.collection) {
+      throw new Error('Please provide a collection.');
+    }
+
+    this.$obContainer = $('#obContainer');
+    this.throttledScroll = __.throttle(this.onScroll, 100).bind(this);
+    this.$obContainer.on('scroll', this.throttledScroll);
+  },
+
+  onScroll: function() {
+    var untilIndex;
+
+    if (this.paginatedCollection &&
+        this.paginatedCollection.length < this.collection.length &&
+        this.paginatedCollection.length < this.MAX_VIEWS &&
+        this.$el.is(':visible') &&
+        // if we're within 200 pixels of the bottom of the scroll container
+        (this.$obContainer[0].scrollTop >= (this.$obContainer[0].scrollHeight - this.$obContainer[0].offsetHeight) - 200)) {
+      untilIndex = this.paginatedCollection.length + this.VIEWS_PER_BATCH > this.MAX_VIEWS ?
+        this.MAX_VIEWS : this.paginatedCollection.length + this.VIEWS_PER_BATCH;
+      this.paginatedCollection.add(
+        this.collection.models.slice(this.paginatedCollection.length, untilIndex)
+      );
+    }
+  },
+
+  createReviewViews: function(reviewModels) {
+    var $container = $('<div/>');
+
+    reviewModels.forEach((reviewMd) => {
```

Comment:
hey - notice the fat arrow function?

I have on my informal to-do list, integrating Babel into our client so we could use some of the cool ES6 and ES7 features it offers. In the meantime, however, I've discovered that Chrome natively supports a lot of the ES6 stuff, like fat arrow functions in this case. They're kind of cool, because a.) the syntax is a bit more compact, although it does take some getting used to b.) It set 'this' to be the 'this' in the outer scope, i.e. all this 'self = this' stuff is no longer needed.